code/game/objects/items.dm

```diff
@@ -619,7 +629,12 @@ var/global/image/fire_overlay = image("icon" = 'icons/effects/fire.dmi', "icon_s
 		var/itempush = 1
 		if(w_class < 4)
 			itempush = 0 //too light to push anything
-		return A.hitby(src, 0, itempush)
+		//return A.hitby(src, 0, itempush)
```

Comment:
Please do not comment out code, remove it, replace it, don't leave old code behind.